app/controllers/volunteers_controller.rb

```diff
@@ -66,6 +66,37 @@ def remove
     # for volunteer to remove project, perhaps include a note for why they're
     # no longer available?
   end
+  
+  def add_repairs
+    @repairs = Repair.all
+    # @repairs = Repair.find(params[:id])
+    @volunteer = current_user.volunteer
+  end
+  
+ def add_repair_to_volunteer
+  # @repair = Repair.find(params[:id])
+   volunteer_repair = VolunteerRepair.new
+   volunteer_repair.volunteer_id = current_user.volunteer.id
+   volunteer_repair.repair_id = params[:id]
+  volunteer_repair.status = "open"
+   volunteer_repair.save
+   redirect_to add_repairs_volunteers_path
+ end
+ 
+ def remove_repair
+   @volunteer_repair = VolunteerRepair.find(params[:id])
+    @volunteer_repair.volunteer_id = current_user.volunteer.id
```

Comment:
This line is unnecessary, because you are going to destroy the record anyway.

Reply:
By the way, the indentation is inconsistent.  For the rb files you have created or edited, you need to run rubocop -a <filename> to correct the indentation and other problems throughout.